resources/views/auth/login.blade.php

```diff
@@ -84,6 +84,6 @@
     <div class="col-9 image">
         <img src="images/logo_transparent.png" class="img-fluid" alt="">
     </div>
-    <a href="/register" class="col-3 mb-1">Register</a>
+    <a href="{{ url('/register') }}" class="col-3 mb-1">Register</a>
```

Comment:
```suggestion
    <a href="{{ route('register') }}" class="col-3 mb-1">Register</a>
```
It's better to use named routes instead of hardcoded urls.